src/index.html

```diff
@@ -6,9 +6,62 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyoHeader">
+      <div class="moyoLogo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="Moyo Logo">
+        </a>
+      </div>
+      
+      <nav>
+        <ul  class="moyoNavigation">
+          <li class="navElement">
+            <a href="#" class="navLink is-active">Apple</a>
```

Comment:
it would be great to add different links as `#apple` etc. Please check other similar cases.
```suggestion
            <a href="#apple" class="navLink is-active">Apple</a>
```